hologram_dashboard/extension/src/app/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, throwError } from 'rxjs';
+import { catchError, retry } from 'rxjs/operators';
+
+@Injectable()
+export class RequestHandler {
+  url: string
+  constructor(private http: HttpClient) { 
+    this.url = "http://localhost:4200/"
```

Comment:
Instead of hard-coding this url in multiple places, could you declare it just in one place?

Reply:
This is only hardcoded in one place I think